app/models/user.rb

```diff
@@ -0,0 +1,34 @@
+class User < ApplicationRecord
+  has_many :posts
```

Comment:
- [ ] You have specified the `has_many` relationship between a `user` and their `posts` 👏 You still need to create the associations between (`user` - `comments`) and (`user` - `likes`).
Please specify the `foreign_key` as well like this 👇 
```ruby
has_many :posts, foreign_key: 'author_id'
```